client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala

```diff
@@ -78,6 +78,7 @@ object CapiModelEnrichment {
       val isInteractive: ContentFilter = content => content.`type` == ContentType.Interactive
 
       val predicates: List[(ContentFilter, Design)] = List(
+        isInteractive -> InteractiveDesign,
```

Comment:
Whenever we make the decision to bump the specificity of a `Format` aspect e.g. `Design`, we need to add a unit test to make sure that any future changes don't clash.